tests/infer/test_valid_models.py

```diff
@@ -966,7 +980,7 @@ def model():
             assert q.shape == (8, 5, 7, 6)
 
     guide = config_enumerate(model, expand=expand) if enumerate_ else model
-    assert_ok(model, guide, Elbo())
+    assert_ok(model, guide, Elbo(max_plate_nesting=4))
```

Comment:
Does this example not work with plate guessing?

Reply:
This test does not work because there are shape assertions in the model. Those assertions assume the model is being enumerated, and they fail when the model is run via `_guess_max_plate_nesting()`. I couldn't see a way to conditionally enable the assertions.

Reply:
Ahh..I see. Thanks for explaining!